addons/l10n_za/data/res.country.state.csv

```diff
@@ -0,0 +1,10 @@
+"id","country_id:id","name","code"
```

Comment:
In Odoo V12.0 the `res.country.state` for ZA is already added in base. Not sure whether it should be added here again as your pull request will probably be targeted toward master. @oco-odoo should we remove the ZA states here or in base?

Reply:
I have removed the file as it was not being used anyway. Should I modify the PR to target master?